service/controller/releasecycle/resource/app/resource.go

```diff
@@ -87,3 +90,13 @@ func toAppCRs(v interface{}) ([]*applicationv1alpha1.App, error) {
 
 	return appCRs, nil
 }
+
+// ReleaseAppCRName returns the name of the release App CR for the given release cycle.
```

Comment:
```suggestion
// releaseAppCRName returns the name of the release App CR for the given release cycle.
```

Reply:
Yes in the past we've stuffed too much stuff in the key package. I'll keep this in mind for app-operator too.